src/amo/components/Footer/index.js

```diff
@@ -51,6 +52,19 @@ export class FooterBase extends React.Component {
                   {i18n.gettext('Blog')}
                 </a>
               </li>
+              <li>
+                <a
+                  className="Footer-extension-workshop-link"
+                  href={`https://extensionworkshop.com/${makeQueryStringWithUTM(
+                    {
+                      utm_content: 'footer-link',
+                      utm_campaign: '',
```

Comment:
Not sure about it that its correct way or not :/

Reply:
could you use `undefined` or `null`? instead? Maybe `undefined` won't work though.

Reply:
Done. 💯